drivers/scheduler/openshift/openshift.go

```diff
@@ -0,0 +1,100 @@
+package openshift
+
+import (
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	kube "github.com/portworx/torpedo/drivers/scheduler/k8s"
+	"regexp"
+	"time"
+)
+
+const (
+	// SchedName is the name of the kubernetes scheduler driver implementation
+	SchedName = "openshift"
+	// SnapshotParent is the parameter key for the parent of a snapshot
+	SnapshotParent = "snapshot_parent"
+	k8sPodsRootDir = "/var/lib/kubelet/pods"
+	// DeploymentSuffix is the suffix for deployment names stored as keys in maps
+	DeploymentSuffix = "-dep"
+	// StatefulSetSuffix is the suffix for statefulset names stored as keys in maps
+	StatefulSetSuffix = "-ss"
+	// SystemdSchedServiceName is the name of the system service resposible for scheduling
+	// TODO Change this when running on openshift for the proper service name
+	SystemdSchedServiceName = "atomic-openshift-node"
+)
+
+const (
+	statefulSetValidateTimeout   = 20 * time.Minute
+	k8sNodeReadyTimeout          = 5 * time.Minute
+	volDirCleanupTimeout         = 5 * time.Minute
+	k8sObjectCreateTimeout       = 2 * time.Minute
+	k8sDestroyTimeout            = 2 * time.Minute
```

Comment:
as @vinayakshnd pointed, reuse these from the k8s package instead of duplicate constants,